Changelog.md

```diff
@@ -33,6 +34,7 @@ Bugfixes:
  * Code Generator: Fix negative stack size checks.
  * Inline Assembly: Enforce function arguments when parsing functional instructions.
  * Fixed segfault with constant function parameters
+ * Optimizer: Disallow optimizations involving ``MLOAD`` because it changes ``MSIZE``.
```

Comment:
Ah sorry. We could keep the two entries or remove the more low-level entry about assembly.